examples/code/async/search_with_configurable_server/run_search_with_configurable_server.errsh

```diff
@@ -1,9 +1,8 @@
   $ jbuilder build search_with_configurable_server.exe
-%% --non-deterministic
+% --non-deterministic
```

Comment:
it's a bit strange that there is only one `%%` symbol here, seems that a regression in the pretty-printer. I'll have a look.